llvm/lib/DebugInfo/Symbolize/MarkupFilter.cpp

```diff
@@ -698,6 +695,24 @@ void MarkupFilter::reportLocation(StringRef::iterator Loc) const {
   errs() << '\n';
 }
 
+void MarkupFilter::symbolizeFailure(const MarkupNode &Node, const Module *Mod,
+                                    Error SymbolizeError) {
+  if (Mod) {
+    auto add_module_info = [Mod](std::unique_ptr<StringError> SE) -> Error {
+      return createStringError(
+          SE->getMessage() +
+              formatv(" for ELF module #{0:x} \"{1}\"; BuildID=", Mod->ID,
```

Comment:
nit: At least to me, the mixture of concatenation and formatv would be easier to read as a single formatv call.